### PR TITLE
add write_data_and_rebuild_index_simultaneously plan

### PR DESCRIPTION
**Pull request by panda-sheep**
add the following plan;
write some data, then write some data to overwrite the previous data and rebuild index at the same time, check if index is compatible with data.

### CODE REVIEW

File: conf/write_data_and_rebuild_index_simultaneously.json [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+        {
+            "type": "WaitAction",
+            "wait_time_ms": 10000,
+            "depends": [12]
+        },
+        {
+            "type": "WriteCircleAction",
+            "tag": "circle",
+            "col": "nextId",
+            "total_rows": 40000,
```
Comment (critical27) [inline conf/write_data_and_rebuild_index_simultaneously.json:133]: Maybe write another circle or a bigger range circle, if we write the same 40000 rows, hard to tell the index is rebuild from previous circle or this one.

Comment (panda-sheep) [reply conf/write_data_and_rebuild_index_simultaneously.json:133]: The data written twice is different.
The first time write random string data 
The second time I wrote the circle data.

Check whether it is a circle data during `loopup`, not  random string data ,
 which verifies that the data of the rebuild index is the second time write .

Comment (panda-sheep) [reply conf/write_data_and_rebuild_index_simultaneously.json:133]: Please refer to the difference between the two WriteCircleActions on line 104 and line 130

**Review by critical27 — APPROVED** [review]
Well done

**Pull request merged**